### PR TITLE
Run against StyleCI

### PR DESCRIPTION
**Pull request by jbrooksuk**
Code standards are tidied up, by [StyleCI](https://styleci.io).

### CODE REVIEW

File: spec/PUGX/Poser/PoserSpec.php [hunk 1]
```diff
@@ -3,23 +3,22 @@
...
 use PUGX\Poser\Render\SvgFlatRender;
 
 class PoserSpec extends ObjectBehavior
 {
-    function let()
+    public function let()
     {
         $render = new SvgFlatRender();
-        $this->beConstructedWith(array($render));
+        $this->beConstructedWith([$render]);
```
Comment (liuggio) [inline spec/PUGX/Poser/PoserSpec.php]: This is not supported can you fix this?

**Comment by liuggio** [comment]
@jbrooksuk  this is great,
thanks a lot,
the tests are red because the array`[]` notation is not supported on old php version can you fix that,
and rebase please?
Great!

File: spec/PUGX/Poser/PoserSpec.php (hunk 1 above)
Comment (jbrooksuk) [inline spec/PUGX/Poser/PoserSpec.php]: Of course.

**Comment by jmleroux** [comment]
Maybe we could stop php 5.3 support and then we could use the short array notation.
PHP 5.3 is not supported anymore : http://php.net/eol.php

**Comment by liuggio** [comment]
if you need a feature of > 5.3 I'd be very Ok to drop it, but for the short notation is not a good reason IMHO,
maybe we could vote against this? @jmleroux / @garak 

ping  @jbrooksuk

**Comment by jbrooksuk** [comment]
Hey, what do you want me to do here? :)

**Comment by garak** [comment]
I think it's better to keep 5.3 support at least until May.

**Comment by jbrooksuk** [comment]
Why May? :)

**Comment by garak** [comment]
Because in May Symfony 2.3 will reach EOL. So we'll be able to raise requirements to Symfony 2.7 and to PHP 5.5

**Comment by jbrooksuk** [comment]
Why not just raise it now and tag as a new major version? That's what Semver is for, right?

**Comment by liuggio** [comment]
I'll add a new version for good reason maybe a  BC or new feature ...
@jbrooksuk BTW please can you pull and merge and replace the array

**Comment by jbrooksuk** [comment]
You should sign up to [StyleCI](https://styleci.io) and configure yourself, so it runs against each PR :)

**Comment by garak** [comment]
Raising now would imply maintaining a branch for old versions, it's not worth the effort.

**Comment by jbrooksuk** [comment]
> Raising now would imply maintaining a branch for old versions, it's not worth the effort.

Hence versioning...

**Comment by garak** [comment]
We can just do versioning with tags and live happily ever after

**Pull request closed without merging**